packages/apollo-server-express/jest.config.js

```diff
@@ -0,0 +1,3 @@
+const config = require('../../jest.config.base');
+
+module.exports = Object.assign({}, config);
```

Comment:
Same comment on this as the earlier one. (plus the other `jest.config.js`'s I didn't comment on)